src/databricks/labs/lsql/dashboards.py

```diff
@@ -106,10 +120,32 @@ def create_dashboard(self, dashboard_folder: Path) -> Dashboard:
             layout = Layout(widget=widget, position=position)
             layouts.append(layout)
 
-        page = Page(name=dashboard_folder.name, display_name=dashboard_folder.name, layout=layouts)
+        page = Page(
+            name=dashboard_configuration.display_name, display_name=dashboard_configuration.display_name, layout=layouts
```

Comment:
```suggestion
            name=dashboard_configuration.display_name, display_name=dashboard_configuration.display_name, layout=layouts,
```
so that it's reformatted nicely